### PR TITLE
Admin screens

### PR DESCRIPTION
**Pull request by kfoley123**

### CODE REVIEW

**Comment by oakeef** [comment]
probably want to get the sign in one merged and then merge main into this and fix the merge conflicsts

File: Screens/Admin/AdminHome/AdminHome.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            <Text> Admin Home </Text>
+            {/* week view of appointments */}
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: "#fff",
```
Comment (oakeef) [inline Screens/Admin/AdminHome/AdminHome.tsx:16]: not something you need to change in this PR but something you can doin the future is make a file with a bunch of variables in it for like styling Constants
you can have like main backround color, font size etx in there and then import them into all the files so you only need to change them in one place

File: Screens/User/UserView/UserView.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { getFocusedRouteNameFromRoute } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import UserTabs from "../UserTabs/UserTabs";
+import UserProfile from "../UserProfile/UserProfile";
+import UserSettings from "../UserSettings/UserSettings";
+
+const Stack = createStackNavigator();
+
+function getHeaderTitle(route) {
+    const routeName = getFocusedRouteNameFromRoute(route);
```
Comment (oakeef) [inline Screens/User/UserView/UserView.tsx:13]: I thought you had changed this in a previous PR
but you can just return this so it's one line

File: Screens/User/UserView/UserView.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                    headerTitle: getHeaderTitle(route),
+                    headerRight: () => (
+                        <Pressable
+                            onPress={() => navigation.navigate("Profile")}
+                        >
+                            <Image
+                                source={{
+                                    uri: "https://i.pinimg.com/originals/cc/2e/01/cc2e011cc5236801ee8fd6d2fc5dc2c5.jpg",
+                                }}
+                                style={{
```
Comment (oakeef) [inline Screens/User/UserView/UserView.tsx:34]: didn't you move this down below into the styles variable as a class?

File: Screens/User/UserView/UserView.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                            />
+                        </Pressable>
+                    ),
+                    headerLeft: () => (
+                        <Pressable
+                            onPress={() => navigation.navigate("Settings")}
+                        >
+                            <Ionicons
+                                name={"md-settings-outline"}
+                                style={{
```
Comment (oakeef) [inline Screens/User/UserView/UserView.tsx:52]: same as comment above

File: Screens/Admin/AdminView/AdminView.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+import { getFocusedRouteNameFromRoute } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import AdminTabs from "../AdminTabs/AdminTabs";
+import AdminProfile from "../AdminProfile/AdminProfile";
+import AdminSettings from "../AdminSettings/AdminSettings";
+
+const Stack = createStackNavigator();
+
+function getHeaderTitle(route) {
+    const routeName = getFocusedRouteNameFromRoute(route);
```
Comment (oakeef) [inline Screens/Admin/AdminView/AdminView.tsx:13]: I thought you had changed this in a previous PR
but you can just return this so it's one line

File: Screens/Admin/AdminView/AdminView.tsx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                    headerTitle: getHeaderTitle(route),
+                    headerRight: () => (
+                        <Pressable
+                            onPress={() => navigation.navigate("Profile")}
+                        >
+                            <Image
+                                source={{
+                                    uri: "https://i.pinimg.com/originals/cc/2e/01/cc2e011cc5236801ee8fd6d2fc5dc2c5.jpg",
+                                }}
+                                style={{
```
Comment (oakeef) [inline Screens/Admin/AdminView/AdminView.tsx:34]: didn't you move this down below into the styles variable as a class?

File: Screens/Admin/AdminView/AdminView.tsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+                            />
+                        </Pressable>
+                    ),
+                    headerLeft: () => (
+                        <Pressable
+                            onPress={() => navigation.navigate("Settings")}
+                        >
+                            <Ionicons
+                                name={"md-settings-outline"}
+                                style={{
```
Comment (oakeef) [inline Screens/Admin/AdminView/AdminView.tsx:52]: same as above?

File: Screens/Admin/AdminHome/AdminHome.tsx (hunk 1 above)
Comment (kfoley123) [reply Screens/Admin/AdminHome/AdminHome.tsx:16]: 👍🏻

**Review by oakeef — APPROVED** [review]

**Pull request merged**